packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -893,8 +893,8 @@ const ADOT_LAMBDA_LAYER_JAVA_SDK_ARNS: { [version: string]: { [arch: string]: {
   },
   '1.30.0': {
     x86_64: {
-      'af-south-1': 'arn:aws:lambda:af-south-1:901920570463:layer:aws-otel-java-wrapper-amd64-ver-1-30-0:1',
-      'ap-east-1': 'arn:aws:lambda:ap-east-1:901920570463:layer:aws-otel-java-wrapper-amd64-ver-1-30-0:1',
+      //'af-south-1': 'arn:aws:lambda:af-south-1:901920570463:layer:aws-otel-java-wrapper-amd64-ver-1-30-0:1',
```

Comment:
@vasireddy99 while you are at this PR can you please add [ADOT_LAMBDA_LAYER_JAVA_SDK_ARNS](https://aws-otel.github.io/docs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.35.0) reference link to L826 of this file.